### PR TITLE
Update soapie inject labels

### PR DESCRIPTION
**Pull request by Cheukting**
Continuation of #19, update the alt labels that got added to the missing ones

### CODE REVIEW

File: src/app/background/index.js [hunk 1]
```diff
@@ -20,10 +20,17 @@ class ImagePrediction {
...
         const payload = {
           imageUrl: req.url,
-          imageAltText: imageAltText
+          imageAltText: `Soapie sees ${imageAltText}`
+        }
+        this.sendMessage(req.tabId, payload)
+      }else{
+        const payload = {
+          imageUrl: req.url,
+          imageAltText: `Soapie sees nothing.`
```
Comment (daeus) [inline src/app/background/index.js]:
```
imageAltText: `Soapie sees ${imageAltText ? imageAltText : "nothing"}`
```
so no need to copy over the whole section?

Comment (daeus) [reply src/app/background/index.js]: or 
```
imageAltText: `Soapie sees ${imageAltText || "nothing"}`
```
might also work

Comment (Cheukting) [reply src/app/background/index.js]: cool, still not get used to those js short hands but I know what you mean. Will update in a bit

Comment (Cheukting) [reply src/app/background/index.js]: Done! If no more request I will merge this PR

**Comment by daeus** [comment]
Merge! Adding the ticket number to the commit message could make it easier to trace back - not a must but nice to have.

**Comment by Cheukting** [comment]
I linked it to your PR in the comment. There's no ticket for this PR

**Pull request merged**